x-pack/plugins/observability_solution/dataset_quality/server/routes/data_streams/get_degraded_docs.ts

```diff
@@ -86,6 +86,7 @@ export async function getDegradedDocsPaginated(options: {
     response.aggregations?.datasets.buckets.map((bucket) => ({
       dataset: `${type}-${bucket.key.dataset}-${bucket.key.namespace}`,
       percentage: (bucket.degraded.doc_count * 100) / bucket.doc_count,
+      count: bucket.degraded.doc_count,
```

Comment:
could we include this in the API tests? We might need to change them a bit because It's a surprise they didn't failed given that we changed the return type.